src/routes.php

```diff
@@ -7,16 +7,16 @@
 //          3. (optional) array of query string keys to send as parameter to the method
 // e.g route '/item/edit?id=1' will execute $itemController->edit(1)
 return [
-    '' => ['HomeController', 'index',],
-    'home' => ['HomeController', 'index',],
-    'games' => ['GameController', 'gamesPages',],
-    'games/add' => ['GameController', 'add',],
-    'users/add' => ['UserController', 'add',],
-    'games/show' => ['GameController', 'index',],
-    'users/show' => ['UserController', 'index',],
-    'games/edit' => ['GameController', 'edit', ['id']],
+    '' => ['HomeController', 'index'],
+    'home' => ['HomeController', 'index'],
+    'games' => ['GameController', 'gamesPages'],
+    'games/add' => ['GameController', 'add', ['view']],
```

Comment:
No ! You should'nt manage this with a view parameter but instead directly in the view with the privileges of your User